testing/jormungandr-integration-tests/src/common/configuration/secret_model_factory.rs

```diff
@@ -21,8 +21,7 @@ impl SecretModelFactory {
     pub fn serialize(node_secret: &NodeSecret) -> PathBuf {
         let content =
             serde_yaml::to_string(&node_secret).expect("Cannot serialize secret node model");
-        let node_config_file_path = file_utils::create_file_in_temp("node.secret", &content);
-        node_config_file_path
+        file_utils::create_file_in_temp("node.secret", &content)
```

Comment:
This will massively conflict with #2270. But I kept the file structure intact to facilitate merging, so hopefully it won't add much work.